yasa/hypno.py

```diff
@@ -228,6 +232,9 @@ def __init__(self, values, n_stages=5, *, freq="30s", start=None, scorer=None):
         assert isinstance(
             scorer, (type(None), str, int)
         ), "`scorer` must be either None, or a string or an integer."
+        assert isinstance(
+            proba, (pd.DataFrame, type(None))
+        ), "`proba` must be either None or a pandas.DataFrame"
         if n_stages == 2:
             accepted = ["W", "WAKE", "S", "SLEEP", "ART", "UNS"]
             mapping = {"WAKE": 0, "SLEEP": 1, "ART": -1, "UNS": -2}
```

Comment:
A similar hand-holding opportunity is here, where I think it'd be nice to inform the user that they need to specify `n_stages` if they are trying to do specify a hypnogram that has less than 5 _possible_ stages.

```python
>>> yasa.Hypnogram(["S", "W"])
# AssertionError: ['S' 'W'] do not match the accepted values for a 5 stages hypnogram: ['WAKE', 'W', 'N1', 'N2', 'N3', 'REM', 'R', 'ART', 'UNS']
```

Sidenote, and minor opinion, but I think the wording throughout should be "N-stage hypnogram" instead of "N stages hypnogram". It's varied in a few places, but I think stages should be singular and the hyphen should be there. To me it seems clearer that it's referring to the number of potential stages (which for many sleep researchers, having <5 is not really an obvious need).

Reply:
Also I think adding the word "unique" to the property string would be helpful: `<Hypnogram | 2 epochs x 30s (1.00 minutes), 2 unique stages>`

Reply:
See edits!